GMnetENGINE.gmx/objects/obj_htme.object.gmx

```diff
@@ -176,12 +194,15 @@ htme_roomstart();
           <argument>
             <kind>1</kind>
             <string>///Disconnect or kick clients if started
-if (!self.isConnected) exit;
-
-if (self.isServer) {
-    htme_serverShutdown();
-} else {
-    htme_clientDisconnect();
+// Tell client/server that we want to disconnect now
+if htme_disconnectNow()

```

Comment:
New way to disconnect a server or client and go to another room. And the engine will re-init and be clean and fit for fight again.